src/ipc/ipc-common.c

```diff
@@ -282,6 +282,7 @@ int ipc_init(struct sof *sof)
 	list_init(&sof->ipc->comp_list);
 
 #ifdef CONFIG_SOF_TELEMETRY_IO_PERFORMANCE_MEASUREMENTS
+	io_perf_monitor_init();
```

Comment:
I need to find a nice place for this init. This has to go before ipc_init. Not sure if we should consider this platform specific (it does not use memory window like previous performance meas. features).

Reply:
Yeah, it almost like we need an entry point for certain system feature/services and we put them in order withing a global services_init function.

Reply:
@tobonex sof/src/init/init.c ?

Reply:
@kv2019i Moved it to primary_core_init in init.c, seems to work. Thanks!